packages/@vuepress/markdown/__tests__/link.spec.js

```diff
@@ -2,10 +2,11 @@ import { Md } from './util'
 import link from '../lib/link.js'
 import { dataReturnable } from '../index.js'
 
-const mdL = Md().use(link, {
+const externalAttrs = {
   target: '_blank',
   rel: 'noopener noreferrer'
-})
+}
+const mdL = Md().use(link, externalAttrs)
 
```

Comment:
Suggest to reuse it by wrapping a setup function. For example

```js
const EXTERNAL_ATTRS = {
  target: '_blank',
  rel: 'noopener noreferrer'
}
const setup = ({ externalAttrs = EXTERNAL_ATTRS, suffix }) => {
  const mdL = Md().use(link, EXTERNAL_ATTRS, suffix)
  dataReturnable(mdL)

  return mdL
}

// Test case
test('should render xxx correctly', () => {
  const mdL = setup()
   ...
})
```

Reply:
Thanks for the pointer, done with 6f8e1cb384c9d7b97af0a4b8147900c7fa4abe7f.